hadoop-plugin/src/test/groovy/com/linkedin/gradle/hadoopValidator/PigValidator/PigDataValidatorTest.groovy

```diff
@@ -0,0 +1,89 @@
+
+package com.linkedin.gradle.hadoopValidator.PigValidator;
+
+import org.junit.Test;
+import static org.junit.Assert.assertArrayEquals;
+
+public class PigDataValidatorTest {
+
+  @Test
+  public void testDataExtract() {
+
+    String fileText = "--D = load './student_data.txt' USING PigStorage(',') as (id : int, firstname:chararray, lastname:chararray, phone:chararray, city:chararray);\n" +
```

Comment:
Instead of embedding the actual text of the scripts, could you add these Pig scripts to the src/test/pig folder, and then in the unit test code actually read the contents of ${projectDir}/src/test/pig/yourPigScript.pig? You should be able to write a nice helper function to accomplish this